lib/erlef_web/controllers/blog_controller.ex

```diff
@@ -34,7 +39,7 @@ defmodule ErlefWeb.BlogController do
 
   defp fetch_working_group(_), do: nil
 
-  defp list(_name) do
-    []
+  defp list(name) do
+    {:ok, Erlef.Blogs.Repo.for_wg(name)}
```

Comment:
Since `for_wg/1` will at the moment consistently return a list, I think we can take this out of the tuple and just have the call to `for_wg/1`, this would be inline with doing away with `with` addressed in the previous comment. As a side note, embedding this as part of the return value in the tagged tuple was a quick hack I did last week and definitely not something you want to do in general or ever :)  Thoughts?

Reply:
Agree